docs/nlu/components.rst

```diff
@@ -100,60 +310,79 @@ However, the additional token at the end (e.g. ``__CLS__``) contains features fo
 This feature vector can be used in any non-sequence model.
 The corresponding classifier can therefore decide what kind of features to use.
 
+
+.. _MitieFeaturizer:
+
 MitieFeaturizer
 ~~~~~~~~~~~~~~~
 
-:Short: MITIE intent featurizer
-:Outputs: nothing, used as an input to intent classifiers that need intent features (e.g. ``SklearnIntentClassifier``)
+:Short:
+    Creates a vector representation of user message and response (if specified) using the MITIE featurizer.
+:Outputs: ``dense_features`` for texts and responses
 :Requires: :ref:`MitieNLP`
 :Type: Dense featurizer
 :Description:
-    Creates feature for intent classification using the MITIE featurizer.
+    Creates features for entity extraction, intent classification, and response classification using the MITIE
+    featurizer.
 
     .. note::
 
-        NOT used by the ``MitieIntentClassifier`` component. Currently, only ``SklearnIntentClassifier`` is able
-        to use precomputed features.
+        NOT used by the ``MitieIntentClassifier`` component.
 
 :Configuration:
+    The sentence vector, e.g. the vector of the ``__CLS__`` token can be calculated in two different ways, either via
+    mean or via max pooling. You can specify the pooling method in your configuration file with the option ``pooling``.
+    The default pooling method is set to ``mean``.
 
     .. code-block:: yaml
 
         pipeline:
         - name: "MitieFeaturizer"
+          # Specify what pooling operation should be used to calculate the vector of
+          # the __CLS__ token. Available options: 'mean' and 'max'.
+          "pooling": "mean"
 
 
+.. _SpacyFeaturizer:
 
 SpacyFeaturizer
 ~~~~~~~~~~~~~~~
 
-:Short: spacy intent featurizer
-:Outputs: nothing, used as an input to intent classifiers that need intent features (e.g. ``SklearnIntentClassifier``)
+:Short:
+    Creates a vector representation of user message and response (if specified) using the spaCy featurizer.
```

Comment:
but text and response is different, two different type of data

Reply:
ok what i mean is that in some places we say "user message" and "response", and in other places "text" and "response". Are those intentionally distinct? If yes, what does "text" mean? Because by general definition, "text" applies to any sort of sentence

Reply:
I replaced "text" with "user message" to make it more clear